terraform/modules/github-ci-bootstrap/README.md

```diff
@@ -160,15 +207,18 @@ Each `service_name` gets its own isolated:
 
 | Name | Description |
 |------|-------------|
-| `service_account_email` | Email of the created service account |
-| `workload_identity_provider` | Full resource name of the Workload Identity provider |
+| `service_account_email` | Email of the created service account (write-enabled branches) |
+| `service_account_email_pr` | Email of the created service account (read-only, available to any branch) |
```

Comment:
What does `pr` mean?  I would have expected `ro` or some such.

Reply:
Looking below, the names here don't match the names in code (which are totally reasonable)